runtime/balrt.h

```diff
@@ -281,6 +281,11 @@ typedef struct {
     PanicCode panicCode;
 } TaggedPtrPanicCode;
 
+typedef struct {
+    int64_t val;
+    bool overflow;
+} DecToIntResult;
```

Comment:
This isn't specific to Decimal.  Call it something like `IntWithOverflow`.

Reply:
Changed